agent/containers/images/image-test-script.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
+
+# A demonstration of how to use pbench-tool-meister-start with an existing
+# Redis server, and one or more remote
```

Comment:
The script only allows for a single remote; although that could be changed by prompting in a loop as done for tools.